lib/gui/css/main.css

```diff
@@ -6402,30 +6402,30 @@ body {
 .component-drive-selector-body .list-group-item-text {
   word-break: break-all; }
 
-.modal-drive-selector-modal .list-group {
+.component-drive-selector-body .list-group {
```

Comment:
OK, cool. Can we send another PR to "re-build CSS" and omit these changes from here?